app/helpers/helpers/helpers_file.go

```diff
@@ -8,6 +8,39 @@ import (
 	"strings"
 )
 
+func GetCommand(line string) string {
+	return line[2:]
+}
+
+func IsTestFile(text string) bool {
+	return text == "t"
+}
+
+func CreateGolangTestFile(text string) string {
+	return text[:len(text)-3] + "_test.go"
+}
+
+func ConvertToFiles(text, packageName string) []string {
+	var res []string
+	files := strings.Split(text, " ")
+
+	for _, file := range files {
+		fileTrimmed := strings.TrimLeft(file, " ")
+
+		if fileTrimmed != "" {
```

Comment:
move to guardians
write
```go
if len(fileTrimmed)==0{continue}
```
also, use len and not "" as len becomes a hook as it can be used for slices and maps

Reply:
addressed in https://github.com/TudorHulban/GoLayouter/commit/d9c081de52e70f86c4ee0af1d15f3af69343fabf

Reply:
please review your code